models:
  yolo_checkpoint: "yolov8x.pt"

camera_color: /camera/image_raw
camera_depth: /camera/depth/image_raw
camera_info:   /camera/camera_info 

perception:
  detection_confidence_threshold: 0.55      # adjust to taste
  device_preference: "cuda"                 # or "cpu"
  camera_frame: "realsense_link_optical"
  base_frame: "map"
  depth_sigma: 15.0
